worksheet_B/Mandelbrot.cpp

```diff
@@ -3,6 +3,125 @@
 
 #include "stdafx.h"
 
+
+//hsv to rgb converter https://www.programmingalgorithms.com/algorithm/hsv-to-rgb?lang=C%2B%2B
+class RGBvalue
+{
+public:
+	unsigned char R;
+	unsigned char G;
+	unsigned char B;
+
+	RGBvalue(unsigned char r, unsigned char g, unsigned char b)
+	{
+		R = r;
+		G = g;
+		B = b;
+	}
+
+	bool Equals(RGBvalue rgb)
+	{
+		return (R == rgb.R) && (G == rgb.G) && (B == rgb.B);
+	}
+};
+
+class HSV
+{
+public:
+	double H;
+	double S;
+	double V;
+
+	HSV(double h, double s, double v)
+	{
+		H = h;
+		S = s;
+		V = v;
+	}
+
+	bool Equals(HSV hsv)
+	{
+		return (H == hsv.H) && (S == hsv.S) && (V == hsv.V);
+	}
+};
+
+static RGBvalue HSVToRGBconverter(HSV hsv) {
```

Comment:
Many programmers (myself included) prefer to make function names start with a verb -- so for example I would call this `convertHSVToRGB`